km_api/know_me/models.py

```diff
@@ -550,6 +570,44 @@ def __str__(self):
         """
         return self.text
 
+    def has_object_read_permission(self, request):
+        """
+        Check read permissions on the instance for a request.
+
+        Args:
+            request:
+                The request to check permissions for.
+
+        Returns:
+            bool:
+                ``True`` if the requesting user owns the km_user the
+                instance belongs to and ``False`` otherwise.
+        """
+        profile_item = self.list_content.profile_item
+        topic = profile_item.topic
+        profile = topic.profile
+        km_user = profile.km_user
+        return request.user == km_user.user
```

Comment:
Rather than specifically requiring that the user owns the object, we should defer the check to the entry's parent `ListContent` instance (`self.list_content.has_object_read_permission(request)`). This will require adding the `IsAuthenticatedMixin` to the list content class, and implementing the `has_object_read_permission` method on it. That method will simply defer to its parent `ProfileItem` instance.

I know that's really wordy, so let me know if it's confusing at all.